ui/src/components/AppBar/AppBar.tsx

```diff
@@ -2,12 +2,20 @@ import React from 'react'
 import { ViewNavigation } from '../ViewNavigation/ViewNavigation'
 import UserMenu from './components/UserMenu/UserMenu'
 import styles from './AppBar.module.css'
+import { useSelector } from 'react-redux'
+import { AppState } from '../../interfaces/storeSlices'
+import { WidgetTypes } from '@tesler-ui/schema'
+import { Layout } from 'antd'
 
-export const AppBar: React.FC = () => {
-    return <header className={styles.container}>
-        <ViewNavigation />
-        <UserMenu/>
-    </header>
+function AppBar() {
+    const widgets = useSelector((state: AppState) => state.view.widgets)
+    const showTabs = widgets?.some(i => i.type === WidgetTypes.SecondLevelMenu)
+    return (
+        <Layout.Header className={styles.container}>
+            <div>{showTabs && <ViewNavigation />}</div>
```

Comment:
Why complicated AppBar for demo? The space for navigation is always there, if there is no tabs `<ViewNavigation />` should handle it itself;

Also excessive `<div />` wrapper

Reply:
@Dergash according to [design](https://www.figma.com/file/xRtm3lEHwsLzP7x1Nyma5E/Tesler?node-id=94%3A45) navigation tabs are not always present.

Reply:
And if tabs not present on specific view, `<ViewNavigation />` should handle itself, there is no reason to split this logic between `<AppBar />` and `<ViewNavigation />`